integration/rabbitmq/e2e/subscribe.e2e-spec.ts

```diff
@@ -32,6 +32,13 @@ const fanoutHandler = jest.fn();
 
 @Injectable()
 class SubscribeService {
+  @RabbitSubscribe({
+    name: 'handler1',
+  })
+  handleSubscribeByName(message: object) {
```

Comment:
Does this handler actually get hit during the test? I would have expected to see more changes to this file including adding a `handlers` config to the module as well as a test that actually publishes such this method would handle the message

Reply:
I haven't written the tests yet. You can ignore this change for now.